skimage/measure/__init__.py

```diff
@@ -1,3 +1,8 @@
+"""
+Tools to measure image metrics and features.
```

Comment:
@Br-Al thank you for submitting a contribution!

I'm just afraid that this description could be too vague, considering that we have another submodule called `metrics`... Looking at https://scikit-image.org/docs/dev/api/skimage.measure.html, I would suggest something more along the lines of "Tools to measure features and compute metrics of (or within) an nD image."

Reply:
Hi @mkcor thank you for your feedback. I will update the docstring according to your comment.